### DISCUSSION TITLE
Removing the "- mpv" appended at the end of the title.

### DISCUSSION THREAD

**Discussion by roxaos** [mpv-player/mpv#14398] [Q&A] [answered]
![image](https://github.com/mpv-player/mpv/assets/47874327/b1aa645d-6ff3-40fd-b98e-ec8ffb3dfb51)

Whenever I run mpv through streamlink every mpv window has a title with "- mpv" appended to the end of whatever I set the title of the stream to. How would I go about removing this to _only_ display the title?

**Comment by llyyr** [comment, marked as answer]
set `title="${?media-title:${media-title}}${!media-title:No file}"`

**Reply by roxaos** [reply]
Where exactly would I make this change? I tried going through the terminal and it didn't work.

`mpv-x86_64>mpv --title="${?media-title:${media-title}}${!media-title:No file}"`

Outputs the basic options and nothing changed after launching mpv via streamlink again.

![image](https://github.com/mpv-player/mpv/assets/47874327/49f80dc7-bd5c-443d-929a-2bb4b9949c81)

**Reply by hooke007** [reply]
If you do not know how to escape, you can firstly try to set it in mpv.conf

**Reply by roxaos** [reply]
> If you do not know how to escape, you can firstly try to set it in mpv.conf

There was no mpv.conf to be found after installation. Looking around I was told that the config was "inside" the .exe which is why I'm trying to go through the command line.

**Reply by hooke007** [reply]
Well. Now i know you haven't read the manual at all.
https://mpv.io/manual/master/#configuration-files

No one will use mpv with cli interface on Windows usually.

**Reply by roxaos** [reply]
First thing I did was read the manual, but I very well may have missed something as I'm not experienced with this, at all, hence why I'm here.

As I said the mpv.conf was not present in any of the directories listed. Running `echo %APPDATA%\mpv\mpv.conf` output a directory with an empty "mpv" folder. 

But in any case, after creating the mpv.conf myself and using the provided command, it's now doing what I want, so much appreciated.

Thank you for your time.

**Reply by hooke007** [reply]
> As I said the mpv.conf was not present in any of the directories listed.

The file is created manually.
